misc/material-migration/migrate-material.rb

```
@@ -0,0 +1,1537 @@
+    else
+      json_block[:attributes] = {
+        'content': new_content,
+        'dropCap': false,
+      }
+    end
+  end
+end
+
+if ARGV.empty?
+  raise ArgumentError, 'Usage: ruby migrate-material.rb <path>, where path is the directory where the material is located.'
+  exit
```

RuboCop (1.87.0)

[warning] Unreachable code detected.

(Lint/UnreachableCode)